packages/os/bootstrap-containers-tmpfiles.conf

```diff
@@ -1,3 +1,3 @@
 d /etc/bootstrap-containers 0750 root root -
```

Comment:
I think we also need to add filecon entries in fs.cil and typetransition in rules.cil, like host-containers.